llvm/test/CodeGen/AVR/branch-relaxation-long.ll

```diff
@@ -7,6 +8,15 @@
 ; CHECK: [[BB1]]:
 ; CHECK: nop
 ; CHECK: [[BB2]]:
+
+; AVR2-LABEL: relax_to_jmp:
+; AVR2: cpi     r{{[0-9]+}}, 0
+; AVR2: brne    [[BB1:.LBB[0-9]+_[0-9]+]]
+; AVR2: rjmp    [[BB2:.LBB[0-9]+_[0-9]+]]
+; AVR2: [[BB1]]:
+; AVR2: nop
+; AVR2: [[BB2]]:
```

Comment:
I think a comment explaining why `rjmp` is intended would be a good idea, to help future people working on this code.

Reply:
Done. More comments is added.